tests/generation/SettingsTest.py

```diff
@@ -42,6 +42,29 @@ def test_write_info(self):
 
         self.assertEqual(result_comment, line)
 
+    def test_allow_complete_section_mode_with_ignore_globs(self):
+        project_dir = "/repo"
+        project_files = ['/repo/hello.html']
+        ignore_globs = ["style.css"]
+        used_languages = list(get_used_languages(project_files))
+        relevant_bears = filter_relevant_bears(
+            used_languages, self.printer, self.arg_parser, {})
+
+        res = generate_settings(
+            project_dir, project_files, ignore_globs, relevant_bears, {}, True)
+
+        bears_list = res["all.HTML"]["bears"].value.replace(" ", "").split(",")
+
+        files_list = res["all.HTML"]["files"].value.replace(" ", "").split(",")
+
+        self.assertEqual(
+            ['HTMLLintBear', 'coalaBear', 'BootLintBear',
+             'LicenseCheckBear', 'SpaceConsistencyBear', 'KeywordBear',
+             'LineLengthBear', 'DuplicateFileBear'].sort(),
+            bears_list.sort())
+
+        self.assertEqual(['**.html'], files_list)
+
```

Comment:
This still doesn't check if the correct `ignore` field is created in the section. Please test this :)

Reply:
Also, just to confirm there's nothing fishy happening behind the hood, add a test case with wildcards in `ignore_globs` (like `**.css`) and assert if the right files are being added to `ignore` field.

Reply:
I don't understand what is meant by that

Reply:
adding globs won't work, I have opened a separate issue for that

Reply:
it is just going to add `**.css` directly to the `ignore` field

Reply:
Verify the `ignore` field contents in the `res` object here. See https://github.com/coala/coala-quickstart/pull/209/files/7a8194e451fb5c904f13aacc086b74bbc59d4c2c#diff-166d3555d5d00b7ce1b006765d440383R124

Reply:
> adding globs won't work, I have opened a separate issue for that

Ah yeah, that's fine then.

Reply:
if I run that individual test, it shows its passed but how do I verify if the correct `ignore` field is generated?

Reply:
I gave you the hint https://github.com/coala/coala-quickstart/pull/209/files/7a8194e451fb5c904f13aacc086b74bbc59d4c2c#r158486786 ;)

Reply:
you need to use `self.assertEqual`

Reply:
thanks I get what you are saying now